Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_PackageShowView.1.html

```diff
@@ -5,9 +5,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <meta charset="UTF-8"/>
     <meta property="og:site_name" content="The Swift Package Index"/>
-    <link rel="canonical" href="http://localhost:8080/"/>
-    <meta name="twitter:url" content="http://localhost:8080/"/>
-    <meta property="og:url" content="http://localhost:8080/"/>
+    <link rel="canonical" href="http://localhost:8080/Alamo/Alamofire"/>
+    <meta name="twitter:url" content="http://localhost:8080/Alamo/Alamofire"/>
+    <meta property="og:url" content="http://localhost:8080/Alamo/Alamofire"/>
```

Comment:
Oh wow, did we really have wrong canonical URLs on all our package pages this whole time?! 😅